docs/user/dashboard/make-dashboards-interactive.asciidoc

```diff
@@ -23,78 +23,125 @@ data-type="inline"
+. If you are creating an *Options list*, specify if you want to *Allow multiple selections in dropdown* and *Run past timeout*.
 
-. If you are adding a *Range slider*, enter the *Step Size* and *Decimal Places*.
+. Click *Save and close*.
 
-. Click *Options*, then configure the following:
+[float]
+[[filter-the-data-with-options-lists]]
+==== Filter the data with Options lists
+Filter the data with one or more options that you select.
 
-* *Update {kib} filters on each change* &mdash; When selected, all interactive inputs create filters that refresh the dashboard. When unselected, {kib} filters are created only when you click *Apply changes*.
+. Open the Options list dropdown.
 
-* *Use time filter* &mdash; When selected, the aggregations that generate the options list and time range are connected to the <<set-time-filter,time filter>>.
+. Select the options for the data you want to display on the dashboard.
++
+The dashboard displays only the data for the options you selected.
 
-* *Pin filters for all applications* &mdash; When selected, all filters created by interacting with the inputs are automatically pinned.
+. To clear the selection, click image:images/dashboard_controlsClearSelections_8.3.0.png[The icon to clear all selected options in the Options list].
 
-. Click *Update*.
+. To display only the options you selected in the dropdown, click image:images/dashboard_showOnlySelectedOptions_8.3.0.png[The icon to display only the options you have selected in the Options list].
 
 [float]
-[[save-the-controls-panel]]
-==== Save and add the panel
+[[filter-the-data-with-range-sliders]]
+==== Filter the data with Range sliders
+Filter the data within a specified range of values.
+
+. On the Range slider, click a value.
+
+. Move the sliders to specify the values you want to display.
++
+The dashboard displays only the data for the range of values you specified.
+
+. To clear the specified values, click image:images/dashboard_controlsClearSelections_8.3.0.png[The icon to clear all specified values in the Range slider].
 
-Save the panel to the *Visualize Library* and add it to the dashboard, or add it to the dashboard without saving.
+[float]
+[[configure-controls-settings]]
+==== Configure the controls settings
 
-To save the panel to the *Visualize Library*:
+. In the dashboard toolbar, click *Controls*, then select *Settings*.
 
-. Click *Save to library*.
+. On the *Control settings* flyout, configure the settings:
 
-. Enter the *Title* and add any applicable <<managing-tags,*Tags*>>.
+* *Label position* &mdash; Specifies where the label appears on the control.
 
-. Make sure that *Add to Dashboard after saving* is selected.
+* *Validate user selections* &mdash; When selected, any selected option that results in no data is ignored.
 
-. Click *Save and return*.
+* *Chain controls* &mdash; When selected, any selected options in one control narrow the available options in the next control.
```

Comment:
Wording nit: should it be "in one control **narrows** the available options"? Or maybe "in one control **will narrow** the available options"